Source/WebKit/WebProcess/WebCoreSupport/WebFrameLoaderClient.cpp

```diff
@@ -73,6 +73,9 @@
 #include <WebCore/Frame.h>
 #include <WebCore/FrameLoader.h>
 #include <WebCore/FrameView.h>
+#if ENABLE(FULLSCREEN_API)
```

Comment:
The conditional headers come *after* the other headers (with a blank line before them), not in the middle of the other headers.